tests/e2e/test_variables.py

```diff
@@ -0,0 +1,78 @@
+"""
+Module with tests for the use of variables
+"""
+import logging
+import subprocess
+
+import pytest
+from pytest_check import check
+
+from . import utils
+
+LOGGER = logging.getLogger(__name__)
+DEFAULT_TIMEOUT = 15
+
+
+@pytest.mark.e2e
+def test_variables_sanity(update_environment):
+    """
+    Execute a rulebook that requires multiple variables to be defined
+    at runtime. Variable values are supplied through environment variables
+    and file variables.
+    """
+
+    env = update_environment(
+        {
+            "SOURCE_EVENT_DELAY": "1",
+            "E2E_AGENT_ID": "86",
+            "E2E_AGENT_NAME": "Maxwell Smart",
+        }
+    )
+
+    rulebook = (
+        utils.BASE_DATA_PATH / "rulebooks/variables/test_variables_sanity.yml"
+    )
+    vars_file = (
+        utils.BASE_DATA_PATH / "extra_vars/test_variables_extra_vars.yml"
+    )
+    cmd = utils.Command(
+        rulebook=rulebook,
+        envvars="SOURCE_EVENT_DELAY,E2E_AGENT_ID,E2E_AGENT_NAME",
+        vars_file=vars_file,
+        debug=True,
```

Comment:
Many things are printed in the debug mode, making the assertions useless. Don't rely in this option here.

Reply:
@Alex-Izquierdo I have thought about this and can't think of another way to validate the rule name. The only other option is to reduce from debug to verbose, which is still quite noisy. Do you have any other ideas?

Reply:
If I remember well, ruleset and rulename are exposed within ansible_eda variable if you use run_playbook. Otherwise, it's fine to me to skip this case. 
Eventlog also exposes it through the websocket, but I think it's making the test unnecessarily complex just for that case.